c2corg_api/models/feed.py

```diff
@@ -0,0 +1,144 @@
+from c2corg_api.models import Base, schema, users_schema, enums
+from c2corg_api.models.area import Area
+from c2corg_api.models.document import Document
+from c2corg_api.models.enums import feed_change_type
+from c2corg_api.models.image import Image
+from c2corg_api.models.user import User
+from c2corg_api.models.utils import ArrayOfEnum
+from sqlalchemy.dialects.postgresql.base import ARRAY
+from sqlalchemy.orm import relationship
+from sqlalchemy.sql.functions import func
+from sqlalchemy.sql.schema import Column, ForeignKey, PrimaryKeyConstraint, \
+    CheckConstraint, Index
+from sqlalchemy.sql.sqltypes import Integer, String, DateTime, Boolean
+
+
+class FilterArea(Base):
+    """Feed filter on areas for a user.
+
+    E.g. with an entry (user_id=1, area_id=2) the user 1 will only
+    see changes in area 2 in their feed (except changes from followed users).
```

Comment:
See https://en.oxforddictionaries.com/usage/he-or-she-versus-they

Instead of writing "his/her" you can write "their".